src/sections/login/loginPage.js

```diff
@@ -38,7 +38,7 @@ export default function LoginPage() {
         phone: "test",
```

Comment:
Please remove "test" phone number

Reply:
User model schema has been updated. Phone value is optional